src/ng-select/items-list.ts

```diff
@@ -41,16 +43,16 @@ export class ItemsList {
         item.selected = true;
     }
 
-    findItem(value, bindValue: string, bindLabel: string): NgOption {
+    findItem(value, bindValue: string): NgOption {
         if (!value) {
             return null;
         }
         if (bindValue) {
-            return this.items.find(x => x[bindValue] === value);
+            return this.items.find(item => item.value[bindValue] === value);
         }
-        const index = this.items.indexOf(value);
+        const index = this.items.findIndex(x => x.value === value);
         return index > -1 ? this.items[index] :
-            this.items.find(x => x[bindLabel] === value[bindLabel])
+            this.items.find(item => item.label === this.resolveNested(value, this._bindLabel))
```

Comment:
Maybe it is better to resolve needed values once when items are loaded so you can have property like value